### ISSUE TITLE
German BOStrab signal A2b in josm preset

### ISSUE THREAD

**Issue by sb12** [rurseekatze/OpenRailwayMap#494]
According to the preset the departure signal A2b is tagged as
``railway:signal:departure=DE-BOStrab:a1``
``railway:signal:departure=DE-BOStrab:a2;off``

I think this should be
``railway:signal:departure=DE-BOStrab:a2b``
``railway:signal:departure=DE-BOStrab:a2b;off``

Or is there any reason for this tagging?

**Comment by rurseekatze** [comment]
Thanks for your feedback. The tagging in the JOSM preset is wrong.

> I think this should be
> ``railway:signal:departure=DE-BOStrab:a2b``
> ``railway:signal:departure=DE-BOStrab:a2b;off``

That is almost OK and similar to https://wiki.openstreetmap.org/wiki/DE:OpenRailwayMap/Tagging_Trams_in_Germany#Abfahren_.28A_2.29. I would suggest ``railway:signal:departure=DE-BOStrab:a`` instead of ``railway:signal:departure=DE-BOStrab:a2b`` because there may be signals that can show A 1 and A 2b. And I guess you missed the `:states` for the second tag.

The wiki documentation (https://wiki.openstreetmap.org/wiki/DE:OpenRailwayMap/Tagging_Trams_in_Germany#Abfahren_.28A_2.29) also needs to be updated. Currently there is only mentioned `a2`, which is wrong because A 2a is just a signal made by staff.

@Nakaner: You created the tagging schema for tram signals – what is your opinion?

**Comment by sb12** [comment]
>  I would suggest ``railway:signal:departure=DE-BOStrab:a`` instead of ``railway:signal:departure=DE-BOStrab:a2b``

Yes that would make even more sense. A 1 probably needs to be changed then as well.

**Comment by sb12** [comment]
@Nakaner: Any feedback on this issue? Or should I just provide a pull request and change the wiki according to @rurseekatze's  suggestion?
